### PR TITLE
add: `bottom-deb`

### PR DESCRIPTION
**Pull request by nezred**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [x] Add maintainer to pacscript

### CODE REVIEW

**Comment by Elsie19** [comment]
Pull from master.

File: packages/bottom-deb/bottom-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name="bottom-deb"
+gives="bottom"
+arch=('amd64' 'arm64' 'armhf')
+repology=("project: ${gives}")
+pkgver="0.9.6"
+homepage="https://clementtsang.github.io/${gives}/"
```
Comment (Elsie19) [inline packages/bottom-deb/bottom-deb.pacscript:6]:
```suggestion
homepage='https://clementtsang.github.io/bottom'
```
Should be single quotes for `homepage`.

File: packages/bottom-deb/bottom-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+url="https://github.com/ClementTsang/${gives}/releases/download/${pkgver}/${gives}_${pkgver}_${CARCH}.deb"
+breaks=("${gives}-git" "${gives}-bin")
+replace=("${gives}")
+pkgdesc="A customizable cross-platform graphical process/system monitor for the terminal."
```
Comment (Elsie19) [inline packages/bottom-deb/bottom-deb.pacscript:10]:
```suggestion
pkgdesc="A customizable cross-platform graphical process/system monitor for the terminal"
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by oklopfer — CHANGES_REQUESTED** [review]
Needs srcinfo generated. Run: 
```bash
./scripts/srcinfo.sh packages/bottom-deb/bottom-deb.pacscript
git add packages/bottom-deb/.SRCINFO
```

**Comment by oklopfer** [comment]
superseded by https://github.com/pacstall/pacstall-programs/pull/6055

**Pull request closed without merging**